src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
```

Comment:
Ideally we could trim it down a bit.  As it's written, I don't think xunit will parallelize the execution of any of these, but waits are an easy thing to parallelize, so if we could make xunit parallelize these (and fix up any issues with regards to how we're managing state, temporary files that could conflict, etc.), we could probably reduce the execution time significantly without sacrificing what tests we run.  If we can't do that, I think it'd be ok to push out a few of the lower priority tests to outer loop.